client/my-sites/stats/stats-subscribers-chart-section/subscribers-navigation-arrows.tsx

```diff
@@ -0,0 +1,77 @@
+import NavigationArrows from '../navigation-arrows';
+
+type SubscribersNavigationProps = {
+	date: Date;
+	period: keyof typeof periodFunctions;
+	quantity: number;
+	onDateChange: ( newDate: Date ) => void;
+};
+
+const isPast = ( date: Date ): boolean => {
```

Comment:
If we want to avoid writing date manipulation functions/tests, we might want to consider using [day.js](https://day.js.org/en/), a 2kB library for date manipulation.